src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/models/Submission.java

```diff
@@ -76,52 +64,40 @@ public Submission(
 
     public UUID getUniversalId() { return universalId; }
 
+    public String getClientAppName() { return clientAppName; }
+
     public FilingPackage getFilingPackage() {
         return filingPackage;
     }
 
-    public Navigation getNavigation() {
-        return navigation;
-    }
-
-    public ClientApplication getClientApplication() { return clientApplication; }
+    public NavigationUrls getNavigationUrls() { return navigationUrls; }
 
     public long getExpiryDate() {
         return expiryDate;
     }
-
-    public AccountDetails getAccountDetails() {
-        return accountDetails;
-    }
-
-    public boolean isRushedSubmission() { return rushedSubmission; }
-
-    public void setAccountDetails(AccountDetails accountDetails) {
-        this.accountDetails = accountDetails;
-    }
-
+    
     public static class Builder {
 
         private UUID id;
         private UUID transactionId;
         private UUID universalId;
-        private AccountDetails accountDetails;
         private FilingPackage filingPackage;
-        private Navigation navigation;
-        private ClientApplication clientApplication;
+        private NavigationUrls navigation;
```

Comment:
nit: name this navigationUrls in this file
```suggestion
        private NavigationUrls navigationUrls;
```